README.md

```diff
@@ -62,7 +62,7 @@ High level items planned for October 2016:
 | tooltip          | Initial version, needs enhancements | [README][18] |              - |
 | ripples          |  Available, but needs to be applied | [README][19] |   [#108][0108] |
 | dialog           |  Started, not yet ready for release |           -  |   [#114][0114] |
-| snackbar / toast |                    Proof-of-concept |           -  |   [#115][0115] |
+| snackbar / toast | Initial version, needs enhancements	| [README][21] |   [#115][0115] |
```

Comment:
It's appears aligned in the github editor, I'm not sure what's up with the diff.. let me see what I can do about it

![image](https://cloud.githubusercontent.com/assets/1303561/19123382/bb3208f4-8b2e-11e6-99c5-6865a7908ec4.png)